ChangeLog.md

```diff
@@ -17,6 +17,10 @@ See docs/process.md for how version tagging works.
 
 Current Trunk
 -------------
+- It is no longer possible to set `wasmMemory` to `INITIAL_MEMORY` at runtime.
```

Comment:
```suggestion
- It is no longer possible to set `wasmMemory` or `INITIAL_MEMORY` at runtime.
```

Reply:
These are potentially controversial changes, as there are use cases for specifying the memory size at runtime.

Perhaps worth discussing on the mailing list first.

If the use cases are strong enough, we may need to support this in some way, like perhaps modifying the wasm binary during startup as I've heard some people already do (can't remember who).